src/forms/checkbox.test.jsx

```diff
@@ -20,52 +21,106 @@ describe('Checkbox', function() {
 		checkbox = null;
 	});
 
-	it('exists', function() {
-		expect(() => TestUtils.findRenderedDOMComponentWithTag(checkboxComponent, 'input')).not.toThrow();
+	describe('basic', function() {
+		it('exists', function() {
+			expect(() => TestUtils.findRenderedDOMComponentWithTag(checkboxComponent, 'input')).not.toThrow();
```

Comment:
Awesome - I'll give that a try. May bug you or Adam for help since I've never used `enzyme` before